### PR TITLE
feat: Rate Limiter token

### PR DESCRIPTION
**Pull request by crisog**
Adds the token to the rate limiter

### CODE REVIEW

File: .github/workflows/production-ap-southeast-2.yml [hunk 1]
```diff
@@ -56,13 +56,15 @@ jobs:
           envkey_LOG_TO_CLOUDWATCH: false
           envkey_AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
           envkey_AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          envkey_COMMIT_HASH: ${{ github.sha }}
```
Comment (adshmh) [inline .github/workflows/production-ap-southeast-2.yml:59]: is this change intentional?

Comment (crisog) [reply .github/workflows/production-ap-southeast-2.yml:59]: These files are auto generated, it was missing in that one file but it is present in all of the other regions. I think it is fine.

**Review by adshmh — APPROVED** [review]
LGTM
